src/containerapp/azext_containerapp/_clients.py

```diff
@@ -196,6 +196,8 @@ def update(cls, cmd, resource_group_name, name, container_app_envelope, no_wait=
         r = send_raw_request(cmd.cli_ctx, "PATCH", request_url, body=json.dumps(container_app_envelope))
 
         if no_wait:
+            # PATCH response body is None
+            r = send_raw_request(cmd.cli_ctx, "GET", request_url)
```

Comment:
why do we need to return response?

Reply:
It will use response to show warning:
`        if "properties" in r and "provisioningState" in r["properties"] and r["properties"]["provisioningState"].lower() == "waiting" and not no_wait:
            logger.warning('Containerapp update in progress. Please monitor the update using `az containerapp show -n {} -g {}`'.format(name, resource_group_name))`